benchmarks/benchmarks.json

```diff
@@ -22,6 +22,12 @@
       "flags": [],
       "extensions": [ "avx2" ]
     },
+    "unpacked_fp32_3x512.mlir": {
```

Comment:
the other "unpacked" benchmark name is after the type, perhaps keep the same style would help.